### ISSUE TITLE
Strengthen & Weaken Armor Enchantments Reversed Effect

### ISSUE THREAD

**Issue by magicono43** [Interkarma/daggerfall-unity#1740]
**Describe the bug**
Due to how the math is done, the Strengthen & Weaken Armor Enchantments have their function reversed. That being that Strengthen armor actually increases player chances of being hit, while Weaken armor reduces player chances of being hit, which is not correct.

**To Reproduce**
1. Find the "CalculateSuccessfulHit" method in the FormulaHelper.cs script. 
2. Add "Debug.LogFormat("Final Hit-Chance = {0}", chanceToHit);" just above line 1015, just before Diceroll100 is called. (This is if you wish to track the hit value through the Unity debug log.)
3. Enchant a weapon or none-armor item, one with "Strengthen Armor", the other with "Weaken Armor"
4. Find a place that you can be hit by an enemy, take off all armor.
5. Check hit value, alternate between equipping the two enchanted items and getting hit to see hit value difference, you will see that while Strength armor is equipped, the value will increase by 5. while the opposite is true with the weaken armor item.

**Expected behavior**
The values should be reversed/effects should be opposite what they currently are.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: Alpha 0.10.21 

**Additional context**
This can be fixed by changing line 968 of FormulaHelper.cs: `armorValue = target.ArmorValues[struckBodyPart] + target.IncreasedArmorValueModifier + target.DecreasedArmorValueModifier;`

Simply change the adds to subtract signs like so: `armorValue = target.ArmorValues[struckBodyPart] - target.IncreasedArmorValueModifier - target.DecreasedArmorValueModifier;`

The other solution would be to change whatever gives the values for these spell effects to the opposite of what they currently are, wherever those values might be held.

**Comment by XJDHDR** [comment]
@magicono43 
Would you be willing to test another thing? When you equip and unequip those two enchanted items, what happens to the armour numbers around your character's paper doll? Does the `Strengthen Armor` enchantment makes these numbers go up or down?

**Comment by magicono43** [comment]
> @magicono43
> Would you be willing to test another thing? When you equip and unequip those two enchanted items, what happens to the armour numbers around your character's paper doll? Does the `Strengthen Armor` enchantment makes these numbers go up or down?

When I did test this a few weeks back. Strengthen would make the paperdoll values go up, and the opposite for weaken armor. So that part of it worked, but in the actual formula the numbers were added/subtracted in a way that was causing the opposite effect on the back-end. Strengthen was making it easier to get hit, and weaken was making it harder. 

Also, if you equipped both at the same time the effect would be negated, both on the paperdoll and in the formula.
